README.md

```diff
@@ -67,9 +68,11 @@ If successful, the driver binary (`IntelHaxm.sys`) will be generated in
 ### Build steps
 1. `cd /path/to/haxm/`
 1. `cd darwin/hax_driver/com_intel_hax/`
-1. `xcodebuild -config Release`
+1. `xcodebuild -configuration Debug`
    * Use `-sdk` to override the default macOS SDK version (10.10), e.g.
 `-sdk macosx10.12`.
+   * Use `Release` instead of `Debug` to build an optimized KEXT that is
+suitable for release.
 
 If successful, the kext (`intelhaxm.kext/`) will be generated in
 `/path/to/haxm/darwin/hax_driver/com_intel_hax/build/Release/`.
```

Comment:
Should we change Release in end of path to Debug?

Reply:
Good catch! Will fix.